custom/icds_reports/utils/aggregation_helpers/distributed/agg_awc.py

```diff
@@ -346,7 +350,7 @@ def updates(self):
         DROP TABLE "tmp_child";
         """.format(
             tablename=self.tablename,
-            child_health_monthly="child_health_monthly",
+            child_health_monthly=self.child_temp_tablename,
```

Comment:
Does not this change of making agg_awc independent of child_health needs change in airflow dependency graph?